devnet/vitosha-3/topology.json

```
@@ -1,17 +1,14 @@
 {
+    "host_network": {
+        "name": "NOLUS",
+        "native_currency": {
+            "name": "Nolus",
+            "symbol": "unls",
+            "ticker": "NLS",
+            "decimal_digits": "6"
+        }
```

```suggestion
        "currency": {
            "id": "NLS",
            "native": {
                "name": "Nolus",
                "symbol": "unls",
                "decimal_digits": "6"
            }
        }
```
two-fold: (1) defines the ticker in a way to be checked in JSON for uniqueness, and (B) the native currency definition to match the native currency definitions of the other networks